package.json

```diff
@@ -106,20 +106,6 @@
             "enzyme-to-json/serializer"
         ],
         "clearMocks": true,
-        "transform": {
```

Comment:
i figured that this is not needed anymore while testing https://github.com/sulu/sulu/pull/6231/files#r710073550 🙂